.gitignore

```diff
@@ -16,3 +16,4 @@ tags
 compiler/*.js
 compiler/**/*.js
 src/.metadata.generated.ts
+src/lib/auto_lib_wrapper_names.generated.ts
```

Comment:
We've been using .-prefixes on the other generated filed, can we follow that convention?

Reply:
Oh, oops, forgot the dotfile, thanks for the catch.